arroyo/backends/kafka/consumer.py

```diff
@@ -428,6 +433,10 @@ def poll(self, timeout: Optional[float] = None) -> Optional[Message[KafkaPayload
                 KafkaError.OFFSET_OUT_OF_RANGE,
                 KafkaError._AUTO_OFFSET_RESET,
             ):
+                # In theory this should not happen under auto offset reset.
+                # However just in case it comes around, be explicit about it.
+                if self.__force_offset_reset:
+                    return None
```

Comment:
I don't think this is what you want. 
if you `force_offset_reset` to error you want to error. I think this would have been ok when the `force_offset_reset` config was a bool.